### ISSUE TITLE
Ability to hold a key to walk

### ISSUE THREAD

**Issue by DigitalOutlaw64** [chylex/Better-Controls#32]
I currently have the toggle sprint option enabled and would love if I could just hold shift (my sprint key) to walk instead of toggling it

**Comment by chylex** [comment]
Do you mean holding Shift to temporarily stop sprinting when it's toggled on?

**Comment by DigitalOutlaw64** [comment]
Yeah, holding shift to stop sprinting, resuming sprint when you’re not holding it. I do a lot of parkour so this feature helps a ton. I found a mod that did this a long time ago in 1.12 but it was never updated

**Comment by chylex** [comment]
Sounds neat, I'll add it when I have time.

**Comment by chylex** [comment]
I gave the implementation some thought but it looks complicated, with all of the other features the mod adds. Might tackle it later, but not in the next update.
